dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.facesProvider-4.0.0.Container.feature

```diff
@@ -0,0 +1,9 @@
+# This private feature corresponds to using a JSF-providing feature
+# with just a container and no implementation (e.g. facesContainer-4.0)
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.facesProvider-4.0.0.Container
+singleton=true
+visibility=private
+kind=ga
+edition=core
```

Comment:
kind=noship edition=full, yes?

Reply:
That's very wrong. Fixed.

noship / full